README.md

```diff
@@ -92,6 +92,7 @@ wsa://com.apple.android.music
 | Audible | 3.15.0 | 11 | ✅
 | Aurora Store | 4.1.1 | 12, 11 | ✅
 | Authy | 24.8.5 (139) | 11 | ✅ || Produces warnings about GMS which are safe to ignore.
+| BandLab | 10.45.0 | 13, 12 | 🆖 | Slight audio lag with android 12, it was fixed in microsoft's 2303.40000.3.0 update | Only tested from installing from google play store with google services, otherwise app has no issues whatsoever. It's responsive, and in android 13 there's no audio lag. |
```

Comment:
```suggestion
| BandLab | 10.45.0 | 13, 12 | 🆖 | Slight audio lag with Android 12, it was fixed in Microsoft's 2303.40000.3.0 update | Only tested from installing from Google Play Store with Google services, otherwise app has no issues whatsoever. It's responsive, and in Android 13 there's no audio lag. |
```